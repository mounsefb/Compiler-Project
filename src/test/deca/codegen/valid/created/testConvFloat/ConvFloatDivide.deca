// Description :
//    Teste de la conversion au type float dans les operations arithmetiques
//
// RÃ©sultats :
//     1
//     1.66667e-01
//     6.00000e+00


{
    int a;
    float b;
    float c;
    c = 6.0;
    a = 1;
    b = a/c;
    println(a);
    println(b);
    println(c);

    
}

