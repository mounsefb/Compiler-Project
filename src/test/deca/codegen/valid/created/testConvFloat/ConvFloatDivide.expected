1
1.66667e-01
6.00000e+00
